middleware/sailauth.py

```diff
@@ -132,7 +134,7 @@ def handle_auth(self, env, start_response):
             'x-auth-token': provided_key,
             'x-storage-token': provided_key,
             'x-auth-token-expires': '86400',
-            'x-storage-url': 'http://127.0.0.1:12345/v1/{}'.format(account),
+            'x-storage-url': 'http://127.0.0.1:8080/v1/{}'.format(account),
```

Comment:
I this hard coded on purpose or should this be an env variable?

Reply:
On purpose for now just for rapid development. Once we find out how it'll be deployed we can add a config